aliases/install.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
It's an alias, I but it in the aliases directory. You may notice I've grouped all the commands in that big switch statement together into a directory called alliases